File: C++/456.cpp
#include <iostream>

using namespace std;

// 456 - Tarta Sácher - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy

int main()
{
    int loop = 0;
    cin >> loop;
    for (int i = 0; i<loop; i++) {
        int counter = 1;
        int n = 0;
        int m = 0;
        int r = 0;
        cin >> n;
        cin >> m;
        cin >> r;
        int nm = n*m;
        while (nm < r) {
            counter++;
            nm += n*m;
        }
        cout << counter << endl;
    }
    return 0;
}

File: Java/456.java
import java.util.Scanner;

/**
 *
 * 456 - Tarta Sácher - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int pruebas = sc.nextInt();
        for (int i = 1; i <= pruebas; i++) {
            int contador = 1;
            boolean booleano = false;
            int ancho = sc.nextInt();
            int alto = sc.nextInt();
            int limite = sc.nextInt();
            int anchoalto = ancho * alto;
            while (!booleano) {
                if (anchoalto >= limite) {
                    System.out.println(contador);
                    booleano = true;
                } else {
                    contador++;
                    anchoalto += ancho * alto;
                }
            }
        }

    }

}
